src/player_display.cpp

```diff
@@ -696,8 +696,8 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4" ) );
                     mvwprintz( w_stats, 6, 21, c_magenta, "%+.1lf", get_hit_base() );
                     mvwprintz( w_stats, 7, 1, c_magenta, _( "Ranged penalty:" ) );
                     mvwprintz( w_stats, 7, 23, c_magenta, "%+3d", -( abs( ranged_dex_mod() ) ) );
-                    mvwprintz( w_stats, 8, 1, c_magenta, _( "Throwing penalty per target's dodge: +%d" ) );
-                    mvwprintz( w_stats, 8, 22, c_magenta, "%3d", throw_dispersion_per_dodge( false ) );
+                    mvwprintz( w_stats, 8, 1, c_magenta, _( "Throwing penalty per target's dodge:" ) );
+                    mvwprintz( w_stats, 8, 22, c_magenta, "+%3d", throw_dispersion_per_dodge( false ) );
```

Comment:
shouldn't it be "%+3d" like the one at two lines above?

Reply:
Perhaps, but this differs from the variant, if the substitution was performed in the original code.

Reply:
Although, I think you're right. I will do the same as in the previous line.